### ISSUE TITLE
Ryujin simulation with multiple inlets (Question)

### ISSUE THREAD

**Issue by rl3418** [conservation-laws/ryujin#176]
[test.txt](https://github.com/user-attachments/files/17098159/test.txt)
Hi,

I am trying to implement a ryujin simulation with two inlets. The file attached is the geometry I defined using Gmsh. I want to have two inlets at the top injecting helium at different temperatures. Not exactly sure how to configure the initialvalues in the prm file. It only takes one direction and a single position for the inlet.

**Comment by ejtovar** [comment]
@rl3418 Do you mind posting the geometry figure? I believe you can use the "function" initial state for custom configurations. See the default `prm` file with description for more info.

**Comment by rl3418** [comment]
![image](https://github.com/user-attachments/assets/078ff3f7-c2fb-47a0-840e-35376854bc2d)
Sorry for the late reply, this is how the geometry looks like. I have got two inlets on the top. Is there an example benchmark script that uses the 'function' initial state?

**Comment by ejtovar** [comment]
@rl3418 I don't believe there is an example for the Euler Equations. Here is a simple example you can modify:

```
subsection function
    set density expression    = (x > 0.) ? 1.4 : 1.0
    set pressure expression   = 1.0
    set velocity x expression = 0.0
    set velocity y expression = (x > 0.) ? -1.0 : -5.0
  end
```
Note the use of the ternary operator which can be useful for your test case.

**Comment by tamiko** [comment]
@rl3418 This is not terribly well explained - we should really update our documentation in this regard: The parameter section for initial values looks as follwos::
```
subsection E - InitialValues
  set configuration = function

  set direction     = 1, 0
  set position      = 0, 0

  subsection function
    ...
  end
end
```

Here, we use the following convention: The initial configuration should be centered in the origin (0,0) and oriented in positive x-direction (1,0). We then use the `direction` and `position` arguments to apply a global Galilean transform: The initial configuration is moved so that the origin rests on the `position` coordinate and the initial configuration is rotated so that the x-direction is facing in `direction` (we apply a rotation to the velocity/momentum fields).

So that means you can do the following:
 - add a function subsection with something like `(x > 0.) ? <value for interior> : ((y > 0) ? <value for right inlet> : <value for left inlet>)`
 - now apply a Galilean transform so that `position` is set to the center point between the two upper boundaries of the inlets, and set `direction` to `0, -1` (facing down).
